### PR TITLE
feat: Fremont Familes Email Template for Script Runner

### PR DESCRIPTION
**Pull request by mcgarrye**
# Pull Request Template

## Issue Overview

This PR addresses #issue

- [ ] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Email translations and template for script runner to use to send emails to Fremont Families applicants.

Language translations done using google translate.

## How Can This Be Tested/Reviewed?

I tested by replacing this [call](https://github.com/bloom-housing/bloom/blob/main/api/src/services/application.service.ts#L662-L666) with a call to the new email service function and completing an application.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: api/src/services/email.service.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ import { SendGridService } from './sendgrid.service';
 import { ApplicationCreate } from '../dtos/applications/application-create.dto';
 import { User } from '../dtos/users/user.dto';
 import { getPublicEmailURL } from '../utilities/get-public-email-url';
+import { Application } from 'src/dtos/applications/application.dto';
```
Comment (YazeedLoonat) [inline api/src/services/email.service.ts]:
```suggestion
import { Application } from '../dtos/applications/application.dto';
```

**Review by YazeedLoonat — APPROVED** [review]
LGTM

File: api/src/views/script-runner.hbs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<h1>{{t "t.hello"}} {{> user-name }}</h1>
+<p>
+  {{t "scriptRunner.information"}}
+</p>
+
+<p>
+  {{t "scriptRunner.yourApplicationNumber" application}}
```
Comment (emilyjablonski) [inline api/src/views/script-runner.hbs:7]: Does this need to be something more specific than `application`?

Comment (YazeedLoonat) [reply api/src/views/script-runner.hbs:7]: Hello! its a bit hard to test in here but I tested with my script runner pr and the email ends up looking like: 
image

Comment (YazeedLoonat) [reply api/src/views/script-runner.hbs:7]: if that looks good I think we can merge and I'll put out the pr that lets us test this more easily!

Comment (emilyjablonski) [reply api/src/views/script-runner.hbs:7]: yep yep!

**Pull request merged**